specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2024-03-01-preview/TuningOptions.json

```diff
@@ -0,0 +1,456 @@
+      "properties": {
+        "databaseName": {
+          "type": "string",
+          "description": "Database name."
+        },
+        "schema": {
+          "type": "string",
+          "description": "Schema name."
+        },
+        "table": {
+          "type": "string",
+          "description": "Table name."
+        },
+        "indexType": {
+          "type": "string",
+          "description": "Index type."
+        },
+        "indexName": {
+          "type": "string",
+          "description": "Index name."
+        },
+        "indexColumns": {
+          "type": "string",
+          "description": "Index columns."
+        },
+        "includedColumns": {
+          "type": "string",
+          "description": "Index included columns."
+        }
+      },
+      "description": "Recommendation details for the recommended action."
+    },
+    "IndexRecommendationResourceProperties": {
+      "type": "object",
+      "properties": {
+        "initialRecommendedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Creation time of this recommendation."
+        },
+        "lastRecommendedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The last refresh of this recommendation."
+        },
+        "timesRecommended": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The number of times this recommendation has encountered."
+        },
+        "improvedQueryIds": {
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "format": "int64"
+          },
+          "description": "The ImprovedQueryIds. The list will only be populated for CREATE INDEX recommendations."
+        },
+        "recommendationReason": {
+          "type": "string",
+          "description": "Reason for this recommendation."
+        },
+        "recommendationType": {
+          "type": "string",
+          "description": "Type for this recommendation."
+        },
+        "implementationDetails": {
+          "type": "object",
+          "properties": {
+            "method": {
+              "type": "string",
+              "description": "Method of implementation for recommended action"
+            },
+            "script": {
+              "type": "string",
+              "description": "Implementation script for the recommended action"
+            }
+          },
+          "description": "Stores implementation details for the recommended action."
+        },
+        "analyzedWorkload": {
+          "type": "object",
+          "properties": {
+            "startTime": {
+              "type": "string",
+              "format": "date-time",
+              "description": "Workload start time."
+            },
+            "endTime": {
+              "type": "string",
+              "format": "date-time",
+              "description": "Workload end time."
```

Comment:
You should add in the description the format

Reply:
Updated description to include format